examples/modern_data_stack_assets/modern_data_stack_assets/assets.py

```diff
@@ -23,10 +23,13 @@
 )
 
 
-@asset(compute_kind="python")
-def order_forecast_model(daily_order_summary: pd.DataFrame) -> Any:
+@asset(
+    compute_kind="python",
+    ins={"daily_order_summary": AssetIn(namespace="public", input_manager_key="numpy_io_manager")},
+)
+def order_forecast_model(daily_order_summary: np.ndarray) -> Any:
     """Model parameters that best fit the observed data"""
-    df = daily_order_summary
+    df = pd.DataFrame(daily_order_summary, columns=["order_date", "total_value", "num_orders"])
```

Comment:
tiny nitpick here, but right now we take in daily_order_summary as a numpy thing, then convert it to pandas, then convert it right back to a numpy thing again on line 35, which is pretty roundabout.

this is actually a perfect usecase for input managers (we want the thing as numpy in our calculations, not pandas, so it's great that we can load the input as pandas in some places and numpy in others), so I think it would be best if we did not do this conversion to pandas at all.

Instead we can slightly change the optimize.curve_fit call to just work directly with numpy. I believe that would look like `f=model_funcm xdata=daily_order_summary[0, :], ydata=daily_order_summary[2, :], p0=[10, 100]` (but I might have gotten the axes flipped).